docs/management/advanced-options.asciidoc

```diff
@@ -78,6 +78,7 @@ Markdown.
 `notifications:lifetime:error`:: Specifies the duration in milliseconds for error notification displays. The default value is 300000. Set this field to `Infinity` to disable error notifications.
 `notifications:lifetime:warning`:: Specifies the duration in milliseconds for warning notification displays. The default value is 10000. Set this field to `Infinity` to disable warning notifications.
 `notifications:lifetime:info`:: Specifies the duration in milliseconds for information notification displays. The default value is 5000. Set this field to `Infinity` to disable information notifications.
+`metrics:max_buckets`:: Used for calculating automatic intervals in Time Series Visual Builder visualizations, this is the maximum number of buckets to represent.
```

Comment:
Isn't it the opposite of  "automatic"? This error arises when people are NOT using "auto".

Maybe something like: "The maximum numbers of buckets that cannot be exceeded. For example, this can arise when the user selects a short interval like (e.g. 1s) for a long time period (e.g. 1 year)".

I dunno..